config/database.yml

```diff
@@ -0,0 +1,31 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
```

Comment:
As the project requirement make sure that your project has Postgres database set up.

<img width="971" alt="image" src="https://user-images.githubusercontent.com/34578631/178715670-43f6eb31-f0d1-4562-b69f-1199e4148493.png">